osbenchmark/resources/docker-compose.yml.j2

```diff
@@ -39,7 +39,7 @@ services:
     networks:
       - opensearch-net
     healthcheck:
-          test: curl -f http://localhost:{{http_port}} -u admin:admin --insecure
+          test: curl -f http://localhost:{{http_port}} -u admin:myStrongPassword123! --insecure
```

Comment:
This command would fail if os_version < 2.12.0. Not sure how this file is used/how we can conditionally switch this password. Can maintainers help out here?

Reply:
@derek-ho Off the top of my head, there's a couple ways that we could get around this. Others might have better ideas
1. You can add an additioonal method that checks the `os_version` variable that's being passed through and if it's < 2.12.0, use `admin`. You'll need to make that additional check in [this section of provisioner.py](https://github.com/opensearch-project/opensearch-benchmark/blob/b861175205a54725c3d7fbbeed02d9be834aa986/osbenchmark/builder/provisioner.py#L503-L505) and  [this section of docker_installer.py ](https://github.com/opensearch-project/opensearch-benchmark/blob/b861175205a54725c3d7fbbeed02d9be834aa986/osbenchmark/builder/installers/docker_installer.py#L109-L112)
2. I believe you could also leverage a library such as`yq` https://stackoverflow.com/questions/76526508/replace-a-block-of-yaml but that would require upkeep with this library and adding it in `setup.py`